jenkinsfile-mode.el

```diff
@@ -236,10 +236,66 @@ Run this manually when editing this file to get an updated the list of keywords.
 (defvar jenkinsfile-mode-font-lock-defaults
   (append groovy-font-lock-keywords jenkinsfile-mode-font-lock-keywords))
 
+(defun jenkinsfile-pipeline-step-compeletion-at-point ()
```

Comment:
Can this and the other methods be prefixed with `jenkinsfile-mode-`?

Reply:
Done.